themes/default/content/blog/cloud-engineering-summit-manage-track/index.md

```diff
@@ -0,0 +1,37 @@
+
+Taking a higher-level view, Quintessence Anx will be presenting *DevSecOps and Secure Incident Response*. The introduction of security to the DevOps equation means some shifts in the way we think about the software development lifecycle, and Quintessence is uniquely qualified to talk about why that’s true. A look inside PagerDuty’s 14 Step Secure Incident Response process promises to be interesting for anyone involved in either development or operations.
+
+I don't think we have the ability to predict the future, but J. Paul Reed wonders if we can develop the ability to forecast incidents just as we have learned to have staggeringly accurate weather short-term forecasts. With *Forecasting the Future: Creating a Radar for Risk*, Paul will share with us how Netflix uses Risk Radar to collect and make sense of risk in their sociotechnical systems.
+
+When we talk about infrastructure as code, there's more to it than just the code! Fariba Khan and Stephen Van Gordon will take us on a journey with *Lifecycle of a Pulumi Program* to share how they leverage a custom state backend with SSO, RBAC, and programmatically configurable pipelines powered by CI/CD tooling and the Pulumi Automation API to drive IaC at their organization.
+
+Snowflake’s Charles Xu will be here to talk about *Day-two Operation of Multi-cloud Kubernetes and Vault*, a very cool overview of Snowflake’s Kubernetes infrastructure and how Pulumi helps, from blue-green updates, to managing Hashicorp Vault, to rotating TLS certificates. This is a pretty complex topic, so I’m very interested to see where it goes!
+
+Gremlin's Jason Yee comes to us with *Trailblazers: exploration, discovery, & navigating failure* - a set of topics that I personally find fascinating and essential! I think we are constantly in a state of discovery and understanding of our complex systems, and learning how to explore them - and work through the inevitable failures - helps us build and operate even more reliable systems.
+
+Rounding things out is the always incredible Tim Banks, with *Toxic Velocity: Speed Kills*. In this talk, Tim is going to cover the various stressors that put pressure on both organizations and individuals, how they affect us, and what it looks like when things start to spiral out of control. I’m excited for this one because while crashes do happen, we’d all like them to happen less often (or not at all!), and being able to identify some of these things before they become a real problem is better for our businesses and our people.
+
+[Explore all of the rest of the great sessions](https://www.pulumi.com/cloud-engineering-summit/sessions/), and don't forget to register [here](https://www.pulumi.com/cloud-engineering-summit/)!
```

Comment:
```suggestion
[Explore all of the rest of the great sessions](https://www.pulumi.com/cloud-engineering-summit/sessions/), and don't forget to [register for the cloud engineering summit](https://www.pulumi.com/cloud-engineering-summit/)!
```

or you could also just use register as the link text too.

ive been trying to get us to stop using here, click here, learn more type link text because it makes it more difficult to scan things quickly. its also not accessible for folks who use screen readers as they often navigate by links and quickly get the context they need. in general its best to use link text that is fully understandable outside of the text that it is within.

ref: https://webaim.org/techniques/hypertext/link_text

Reply:
I agree! I try to avoid that but missed that one - I like contextual link vs "click here" much better myself. Ty!